API-Lib/src/main/java/de/muenchen/vaadin/demo/apilib/domain/NoKeyWord.java

```diff
@@ -0,0 +1,34 @@
+package de.muenchen.vaadin.demo.apilib.domain;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+/**
+ * Created by arne.schoentag on 18.12.15.
+ */
+@Constraint(validatedBy = CheckKeyWordValidator.class)
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface NoKeyWord {
+
+    String message() default "{de.muenchen.vaadin.demo.keyword.message}";
```

Comment:
nein, ich kann die Annotation sowohl in der gui als auch in dem microservice nutzen